src/pybind/rados/rados.pyx

```diff
@@ -2692,6 +2693,38 @@ cdef class Ioctx(object):
             raise make_ex(ret, "failed to compare %s" % object_name)
         return completion
 
+    @requires(('object_name', str_type), ('xattr_name', str_type), ('oncomplete', opt(Callable)))
+    def aio_rmxattr(self, object_name, xattr_name, oncomplete = None):
+        """
+        Asynchronously delete an extended attribute from an object
+        :param object_name: the name of the object to remove xattr from
+        :type object_name: str
```

Comment:
might want to drop the docstring documenting the parameter types since we are using PEP484 style annotations.